src/region/actions.ts

```diff
@@ -1,20 +1,27 @@
-import { AppThunkDispatch } from '../store';
+import { AppThunkDispatch, AppState } from '../store';
 import { createAnnotationAction } from '../store/annotations';
 import { Rect } from '../@types';
+import { getFileVersionId } from '../store/options';
 
 export type CreateArg = {
     location: number;
     message: string;
     shape: Rect;
 };
 
-export const createRegionAction = (arg: CreateArg) => (dispatch: AppThunkDispatch) => {
+export const createRegionAction = (arg: CreateArg) => (dispatch: AppThunkDispatch, getState: () => AppState) => {
     const { location, message, shape } = arg;
+    const state = getState();
     const newAnnotation = {
         description: {
             message,
             type: 'reply' as const,
         },
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        file_version: {
+            id: getFileVersionId(state) ?? undefined,
```

Comment:
It looks like the selector returns `string | null`. Can we keep that consistent here? Or should we move this to the eventing middleware if we want different behavior?